### PR TITLE
[Host] Include Config.h to make sure READLINE_HAS_WCHAR_T is honoured.

### PR DESCRIPTION
**Pull request by dcci**
<rdar://problem/48401960>

### CODE REVIEW

**Comment by dcci** [comment]
@swift-ci test and merge

**Comment by dcci** [comment]
@adrian-prantl can you please review this one?

File: include/lldb/Host/Editline.h [hunk 1]
```diff
@@ -33,6 +33,8 @@
 #define liblldb_Editline_h_
 #if defined(__cplusplus)
 
+#include "lldb/Host/Config.h"
```
Comment (adrian-prantl) [inline include/lldb/Host/Editline.h:36]: For the record that file has
```
#if defined(__APPLE__)

// This block of code only exists to keep the Xcode project working in the
// absence of a configuration step.
#define LLDB_CONFIG_TERMIOS_SUPPORTED 1

#define LLDB_EDITLINE_USE_WCHAR 1
...
```

**Review by adrian-prantl — APPROVED** [review]
That should do the trick, yes.

File: include/lldb/Host/Editline.h (hunk 1 above)
Comment (adrian-prantl) [reply include/lldb/Host/Editline.h:36]: We'll need to find something cleaner for upstream / llvm.org

**Pull request merged**